### ISSUE TITLE
Output nifti preview after harmonizing

### ISSUE THREAD

**Issue by hajerkr** [rpomponio/neuroHarmonize#40]
Hello @rpomponio  @vnckppl  @AbdulkadirA 

Is the rim around normal after harmonizing? There's strange black halo around the brain.

**Before (original nifti)**:
<img width="160" alt="image" src="https://github.com/rpomponio/neuroHarmonize/assets/13291010/24b8e9d9-fb4c-4af1-804b-28acfbbb3112">

**After (after harmonizing)**
<img width="299" alt="image" src="https://github.com/rpomponio/neuroHarmonize/assets/13291010/f4eae2cd-86da-488f-bd58-5fafed0a8bd1">

Tailrach is failing on all harmonized images. Any input would be appreciated thank you

**Comment by AbdulkadirA** [comment]
@hajerkr Thanks for reporting the issue and demonstrating it with data. The method is intended for co-registered images of "quantitative" (as in tissue probability or RAVENS maps). It is not intended for T1 images where there is no voxel-to-voxel correspondence across images. No amount of co-registration will make the method viable for harmonizing the raw data.

**Comment by rpomponio** [comment]
Agreed with @AbdulkadirA. Harmonization on NIFTI images only supported for RAVENS maps or other template-space maps. Closing the issue.
